job-server/src/main/resources/application.conf

```diff
@@ -14,17 +14,13 @@ spark {
     # Number of job results to keep per JobResultActor/context
     job-result-cache-size = 5000
 
-    jobdao = spark.jobserver.io.JobFileDAO
+    jobdao = spark.jobserver.io.JobSqlDAO
 
     # Automatically load a set of jars at startup time.  Key is the appName, value is the path/URL.
     # job-jar-paths {
     #   test = /path/to/my/test.jar
     # }
 
-    filedao {
```

Comment:
See my comments in issue. Let's not get rid of this section yet, but leave a comment that FileDAO has issues in production and should not be used.    This will make it easier for folks to transition off.